undocked/tools/export_program_info_sample/export_program_info_sample.vcxproj

```diff
@@ -122,19 +100,19 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <AdditionalDependencies>mincore.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
-    <CustomBuildStep>
+    <CustomBuildStep Condition="'$(Platform)'=='x64'">
```

Comment:
Are this build step (and the following several) x64-specific?

Reply:
Yes. The challenge is when cross-compiling for ARM64 on x64 host, we can't execute the ARM64 version of the export tool, but ARM64 can execute the x64 version of the tool.

So, the solution when building the complete project will be to first build x64, then build ARM64.